sycl/unittests/SYCL2020/DeviceAspectTraits.cpp

```diff
@@ -0,0 +1,26 @@
+#include <sycl/sycl.hpp>
+
+#include <helpers/PiMock.hpp>
+
+#include <gtest/gtest.h>
+
+TEST(DeviceAspectTraits, AnyDeviceHasAspect) {
+#define __SYCL_ASPECT(ASPECT, ASPECT_VAL)                                      \
+  constexpr bool CheckAnyDeviceHas##ASPECT =                                   \
+      sycl::any_device_has_v<sycl::aspect::ASPECT>;                            \
+  std::ignore = CheckAnyDeviceHas##ASPECT;
+
+#include <sycl/info/aspects.def>
```

Comment:
As I said above, there are also aspects which are marked deprecated defined in a separate file. They are part of `aspect` enum and therefore should be tested here as well.

Reply:
Added! Thanks for pointing this out.